### PR TITLE
Create the InsightsPage framework

### PR DESCRIPTION
**Pull request by Rijidij**
A skeleton insights page (and index page) that we can add components to as they are developed.

### CODE REVIEW

File: etna/insights/models.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+class InsightsIndexPage(Page):
+    """InsightsIndexPage
+
+    This page lists the InsightsPage objects that are children of this page.
+    """
+    introduction = models.CharField(max_length=200, blank=False)
+
+    def get_context(self, request):
+        context = super().get_context(request)
+        insights_pages = self.get_children().live()
```
Comment (danbentley) [inline etna/insights/models.py:20]: This looks good. 

It's probably worth excluding private pages:

https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html#wagtail.core.query.PageQuerySet.public

Along with the specific page type instances:

https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html#wagtail.core.query.PageQuerySet.specific

File: etna/insights/models.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    subpage_types = ['insights.InsightsPage']
+
+
+class InsightsPage(Page):
+    """InsightsPage
+
+    The InsightsPage model.
+    """
+    introduction = models.CharField(max_length=200, blank=False)
+    content = StreamField(InsightsPageStreamBlock, blank=True, null=True)
```
Comment (danbentley) [inline etna/insights/models.py:37]: Could we rename this attribute to `body`? This keeps it consistent with our other sites and is becoming a convention:

https://docs.wagtail.io/en/stable/reference/streamfield/blocks.html#streamfield-block-reference

**Review by danbentley — CHANGES_REQUESTED** [review]
Thanks for this. It looks good. I like that you've included a `StreamBlock` to be populated with our other component types. 

I've requested a few minor changes, just to keep things consistent.

File: templates/insights/insights_page.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    {% include 'includes/generic-intro--dark.html' %}
+
+    <div class="container">
+        {{ page.content }}
```
Comment (danbentley) [inline templates/insights/insights_page.html:7]: The template still references `content`

File: etna/insights/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.8 on 2021-07-08 13:26
```
Comment (danbentley) [inline etna/insights/migrations/0001_initial.py:1]: Could you squash these migrations down to a single file:

Roll back the insights migration:

`python manage.py migrate insights zero`

Delete the existing migration files

Recreate:
`python manage.py makemigrations insights`

File: templates/insights/insights_page.html (hunk 3 above)
Comment (Rijidij) [reply templates/insights/insights_page.html]: Done

File: etna/insights/migrations/0001_initial.py (hunk 4 above)
Comment (Rijidij) [reply etna/insights/migrations/0001_initial.py]: and done.

**Pull request merged**